go-runtime/compile/schema.go

```diff
@@ -1284,9 +1285,13 @@ func visitType(pctx *parseContext, pos token.Pos, tnode types.Type, isExported b
 		case types.String:
 			return optional.Some[schema.Type](&schema.String{Pos: goPosToSchemaPos(pos)})
 
-		case types.Int, types.Int64:
+		case types.Int:
 			return optional.Some[schema.Type](&schema.Int{Pos: goPosToSchemaPos(pos)})
 
+		case types.Int64:
+			pctx.errors.add(tokenErrorf(1234, underlying.Name(), "int is not a supported type; specify int instead %s", underlying.Name()))
```

Comment:
`int64 is not a supported type; instead, specify int for %q`. And friendly reminder to not forget about replacing `1234` with the position data :)

Reply:
btw if you'd like to see how this error shows up for real, you can try adding it to `failing.go`. There's a test for it in `go-runtime/compile/schema_test.go`.

Reply:
that was placeholder for sure! thanks for catching that. The error message does not need to be reported since the calling code reports a meaningful error when None is returned as the type. I've deleted the `Int64` case entirely.

Reply:
awesome!